functions.php

```diff
@@ -44,15 +54,17 @@ function applyToDB()
         $sql = 'INSERT INTO `records` (`Artist`, `Title`, `Rating`, `Released`, `Condition`)
         VALUES (:Artist, :Title, :Rating, :Released, :Condition)';
         $statement = $db->prepare($sql);
-        $statement->execute([
+        $executed = $statement->execute([
             ':Artist' => $artist,
             ':Title' => $title,
             ':Rating' => $rating,
             ':Released' => $released,
             ':Condition' => $condition
         ]);
-        if ($statement == true) {
+        if ($executed == true) {
             header('Location: index.php');
         }
+    } else {
```

Comment:
Hm, there's a small edgecase here.
If the form is all valid, but it isn't added to the database successfully (which might not be something in your control and I'm not sure you can test it), it won't execute either of the `header()` functions. Since both functions are redirecting you to the same place no matter what happens, rather than using an `else` (which would have made sense if you had an error page, for example!), please call `header('Location: index.php');` after your first if statement, and remove the `else` and the if statement on line 64.